luigi/contrib/kubernetes.py

```diff
@@ -276,6 +281,11 @@ def __verify_job_has_started(self):
 
         # Verify that the pod started
         pods = self.__get_pods()
+        if not pods:
```

Comment:
Not sure I follow the logic here. How would the `sleep` impact the assertion afterwards?

Reply:
I'm with @honnix . Seems like you'd want to re-eval `pods=self.__get_pods()` before asserting

Reply:
Good catch, thanks